npm/nameSpaceController.go

```diff
@@ -395,10 +427,16 @@ func (nsc *nameSpaceController) syncUpdateNameSpace(newNsObj *corev1.Namespace)
 			metrics.SendErrorLogAndMetric(util.NSID, "[UpdateNamespace] Error: failed to add namespace %s to ipset list %s with err: %v", newNsName, labelKey, err)
 			return err
 		}
+		// {IMPORTANT} Same as above order is assumed to be key and then key+val. NPM should only append to existing labels
```

Comment:
Same. Can we resolve this dependency?

Reply:
As we discussed offline, keeping this as it is for now.